tests/foreman/ui/test_activationkey.py

```diff
@@ -1101,12 +1100,46 @@ def test_associate_host(self):
 
         @Assert: Hosts are successfully associated to Activation key
 
-        @Status: Manual
-
         @BZ: 1078676
 
         """
-        pass
+        key_name = gen_string("alpha", 10)
+        with Session(self.browser) as session:
+            make_activationkey(session, org=self.org_name,
```

Comment:
This looks better:

```
make_activationkey(
    session, org=self.org_name, name=key_name, env=ENVIRONMENT)
```

Or

```
make_activationkey(
    session,
    org=self.org_name,
    name=key_name,
    env=ENVIRONMENT
)
```

It the first does not fit in the line length.